Framework/Beamline/src/ComponentInfo.cpp

```diff
@@ -51,6 +53,7 @@ ComponentInfo::ComponentInfo(
       m_detectorRanges(std::move(detectorRanges)),
       m_componentRanges(std::move(componentRanges)),
       m_parentIndices(std::move(parentIndices)),
+      m_assemblyImmediateChildren(std::move(assemblyImmediateChildren)),
```

Comment:
How about `m_children`?

Reply:
I chose this name because I wanted to make the distinction between what is returned by `ComponentInfo::detectorsInSubtree` now `ComponentInfo::detectorsInFullSubtree` and the fact that this only returns children in the next (immediate) tree level. Which could be "Component Assemblies" or leaf nodes.

Reply:
Well, anything that is not in the next tree level are grandchildren, or grand-grand children. I believe "tree" and "children" have a commonly understoof and unique meaning?